cmd/witness/cmd/run.go

```diff
@@ -44,11 +45,27 @@ func RunCmd() *cobra.Command {
 }
 
 func runRun(ro options.RunOptions, args []string) error {
-	signer, err := loadSigner(ro.KeyOptions.SpiffePath, ro.KeyOptions.KeyPath, ro.KeyOptions.CertPath, ro.KeyOptions.IntermediatePaths)
-	if err != nil {
-		return fmt.Errorf("failed to load signer: %w", err)
+	ctx := context.Background()
+	signers, errors := loadSigners(ctx, ro.KeyOptions)
+	if len(errors) > 0 {
+		for _, err := range errors {
+			log.Error(err)
+		}
+		return fmt.Errorf("failed to load signers")
+	}
+
+	if len(signers) > 1 {
```

Comment:
why can loadSigners return multiple if only one is supported?  whats stopping us from supporting multiple signers?